mix.exs

```diff
@@ -60,8 +60,8 @@ defmodule Wallaby.Mixfile do
       {:credo, "~> 0.9", only: [:dev, :test], runtime: false},
       {:bypass, "~> 1.0.0", only: :test},
       {:excoveralls, "~> 0.7", only: :test},
-      {:ex_doc, "~> 0.20", only: :docs},
-      {:inch_ex, "~> 2.0", only: :docs}
+      {:ex_doc, "~> 0.20", only: :dev},
+      {:inch_ex, "~> 2.0", only: :dev}
```

Comment:
Just curious... why remove the `:docs` environment? I first saw this pattern with phoenix and thought it made sense because `ex_doc` and `inch_ex` aren't needed during development, only when generating documentation.

https://github.com/phoenixframework/phoenix/blob/3d964c67c7bf4c282bce36fca57bf990691a222a/mix.exs#L77-L79

Reply:
TBH, I forgot that I had switched the actual dependencies back to `:dev` in this PR, I was thinking I just switched the `preferred_cli_env`. 

I thought this was originally made this way due to an issue with the Elixir version we were using. I don't feel super strongly about it, I can switch it back.

I should also set the `preferred_cli_env` for the `mix inch` tasks so we don't have to worry about setting the env there either.

Reply:
I think this simplifies things a bit? I wouldn't mind if they get downloaded when I'm working on wallaby.

That said, they still won't get downloaded for wallaby users in dev env, right? This would be less ideal, but I'm not sure how that works, I'd have to test it out.

Reply:
Does the env actually change which deps are downloaded or just which ones are loaded?

Reply:
I was just playing around a little and found it doesn't change which deps are downloaded by `mix deps.get` (unless you use `--only dev`). However when I run `mix deps.compile`, if these documentation libraries are in a different MIX_ENV, it won't compile or load these libraries or their dependencies: 

```
$ mix deps.tree ex_doc
ex_doc ~> 0.20 (Hex package)
├── earmark ~> 1.3 (Hex package)
└── makeup_elixir ~> 0.14 (Hex package)
    └── makeup ~> 1.0 (Hex package)
        └── nimble_parsec ~> 0.5.0 (Hex package)

$ mix deps.tree inch_ex
inch_ex ~> 2.0 (Hex package)
├── bunt ~> 0.2 (Hex package)
└── jason ~> 1.0 (Hex package)
```